instruqt-tracks/terraform-intro-gcp/track_scripts/setup-workstation

```diff
@@ -43,9 +43,11 @@ cp ${GITDIR}/exercises/outputs.tf.start ${GITDIR}/outputs.tf
 mv ${GITDIR}/terraform.tfvars.example ${GITDIR}/terraform.tfvars
 
 # Store our project ID as a Terraform env var
-export TF_VAR_project=$INSTRUQT_GCP_PROJECT_GCP_PROJECT_PROJECT_ID
-grep $INSTRUQT_GCP_PROJECT_GCP_PROJECT_PROJECT_ID /root/.bashrc || echo "export TF_VAR_project=\"$INSTRUQT_GCP_PROJECT_GCP_PROJECT_PROJECT_ID\"" >> /root/.bashrc
```

Comment:
Why are we `grep`ping for the value in `/root/.bashrc`? If you needed to grep, it's far safer and more robust to `grep "^export TF_VAR_PROJECT="`. But in any case, this is the track setup script, nothing has added it, so just add it.